lib/elixir/test/elixir/application_test.exs

```diff
@@ -21,6 +21,20 @@ defmodule ApplicationTest do
     assert Application.get_env(:elixir, :unknown, :default) == :default
   end
 
+  test "application information" do
+    started = Application.started_applications
+    assert started |> is_list
```

Comment:
Everyone loves pipes, [even too much](https://twitter.com/lexmag/status/618432396816457728). :smile:
It just feels excessive to use it for a single function call IMO.